frontend/src/components/AdminDash/events/EventDetails.jsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { Modal, ModalHeader, ModalBody, ModalFooter, Button } from 'reactstrap';
+import * as SForm from '../shared/formStyles';
+import PropTypes from 'prop-types';
+import { eventValidator } from './eventHelpers';
+import { createEvent } from '../queries';
+import { Icon } from 'components/Shared';
+
+const Styled = {
+    Button: styled(Button)`
+        position: absolute;
+        align-self: flex-end;
+        background: none;
+        border: none;
+        opacity: 1.0;
+    `,
+
+    FlexHeader: styled.div`
+        display: flex;
+        width: 100%;
+    `,
+
+    EventInfoHeader: styled.div`
+    `,
+
+    h5: styled.h5`
+        color: #969696
+    `
+};
+
+function IconHeading({iconName, headerText, subHeaderText}) {
+    return (
+        <Styled.FlexHeader>
+            {iconName &&
+                <Icon color="grey3" name={iconName} />
+            }
+            <div>
+                <h3>{headerText}</h3>
+                <Styled.h5>{subHeaderText}</Styled.h5>
+            </div>
+        </Styled.FlexHeader>
+    );
+}
+
+// TODO: Probably already exists elsewhere, so import this instead?
```

Comment:
I recommend using [Moment.js](https://momentjs.com/) to make handling dates easier/more streamlined